frontend/src/metabase/admin/permissions/components/DataPermissionsHelp/DataPermissionsHelp.tsx

```diff
@@ -0,0 +1,126 @@
+            />
+
+            <PermissionHelpDescription
+              icon="permissions_limited"
+              iconColor="warning"
+              name={t`Granular`}
+              description={t`The group can only use the query builder to ask questions of specific tables.`}
+            />
+
+            <PermissionHelpDescription
+              icon="database"
+              iconColor="warning"
+              name={t`Impersonated (Pro)`}
+              description={t`Impersonation associates a Metabase group with database-defined roles and their privileges. Metabase queries made by this group will respect the grants given to the database roles. You can use impersonation to give a group access to the native/SQL editor, while restricting the group's access to data based on a specific database role.`}
+            />
+
+            <PermissionHelpDescription
+              icon="eye_crossed_out"
+              iconColor="accent5"
+              name={t`No self-service`}
+              description={t`The group can't use the query builder or drill through existing questions. They also can't see the data in the Browse data section. They can still view questions based on this data, if they have permissions to the relevant collection.`}
+            />
+
+            <PermissionHelpDescription
+              icon="close"
+              iconColor="danger"
+              name={t`Block (Pro)`}
+              description={t`The group can't see questions based on this the data, even if they have collection permissions to view the questions. People in a blocked group would need to be in another group with both collection permissions and data permissions in order to view the item.`}
+            />
+            <Text>{t`Only available in certain Metabase plans.`}</Text>
+          </Stack>
+        </Accordion.Panel>
+      </Accordion.Item>
+
+      <Accordion.Item value="schema-table-level">
+        <Accordion.Control>Schema and table levels</Accordion.Control>
+        <Accordion.Panel>
+          <Stack spacing="1rem" py="1rem">
+            <PermissionHelpDescription
+              icon="check"
+              iconColor="success"
+              name={t`Unrestricted`}
+            />
+
+            <PermissionHelpDescription
+              icon="eye_crossed_out"
+              iconColor="accent5"
+              name={t`No self-service`}
+              description={t`"Unrestricted" and "No self-service permissions" work like they do for databases, except here they're scoped to individual schemas or tables.`}
+            />
+
+            <PermissionHelpDescription
+              icon="permissions_limited"
+              iconColor="brand"
+              name={t`Sandboxed (Pro)`}
+              description={t`Let's you specify row and column-level permissions. Can be set up via user attributes and SSO.`}
+            />
+
+            <Text>{t`Only available in certain Metabase plans.`}</Text>
```

Comment:
I saw the design docs specified having an `Upgrade to Pro` link. Were you waiting to add this to see if product wanted it on both accordions under 'Only available in certain Metabase plans'?